src/Todo.js

```diff
@@ -1,7 +1,45 @@
 'use strict';
 
 class Todo {
-  // write code here
+  constructor() {
+    this.listOfItems = [];
+    this.idNumber = -1;
+  }
+  addItem(titleInput, priorityInput) {
```

Comment:
зачем нам приписывать слово input?
```suggestion
  addItem(title, priority) {
```